File: image-classifier/src/app/kafka_config.py
import json, boto3

secret_arn = "arn:aws:secretsmanager:us-east-1:117819748843:secret:main_kafka_api_key-beAUbS"

secret = json.loads(
        boto3.client("secretsmanager", 'us-east-1')
        .get_secret_value(SecretId=secret_arn)
        ["SecretString"]
)

kafka_api_key = secret["key"]
kafka_api_secret = secret["secret"]
bootstrap_servers = secret["bootstrap_servers"]

File: video-stream-reader/src/test/scala/com/kgmcquate/video/livestream/reader/KafkaConfig.scala
package com.kgmcquate.video.livestream.reader

object KafkaConfig {
  private val secretName = "arn:aws:secretsmanager:us-east-1:117819748843:secret:main_kafka_api_key-beAUbS"
  private val region = "us-east-1"

  private val secrets = SecretsManagerUtil.getSecret(secretName, region)

  val bootstrapServers: String = secrets("bootstrap_servers")
  val kafkaApiSecret: String = secrets("secret")
  val kafkaApiKey: String = secrets("key")

}
